main/src/cgeo/geocaching/apps/navi/MapsMeApp.java

```diff
@@ -72,4 +75,17 @@ public boolean isInstalled() {
         return true;
     }
 
+    private static Activity getActivity(final Context context) {
+        if (context == null) {
```

Comment:
The first if is superfluous. instanceof is always false for null, therefore a null value bypasses the instanceof checks and returns null from the end of the method already.

Reply:
never trust a sample code :-)
thanks for pointing.